general/README.md

```diff
@@ -4,4 +4,102 @@ Guidelines and best practices used by Eventbrite to provide consistency and prev
 
 ## Table of Contents
 
-Coming soon...
+0. [Conditionals](#conditionals)
+
+
+## Conditionals
+
+### Complex conditional expressions
+
+Complex conditional expressions within conditional statements can make understanding code challenging because you need to fully understand the expression in order to be able to process the overall code flow. In addition, in code reviews (or revisiting old code), it's difficult to know if the logic is correct without knowing what state the conditional expression is trying to represent.
+
+To make things easier, store complex conditional expressions in state variables:
+
+```js
+// good
+var shouldContinue = !options || (options.continue && options.hasWon);
+
+if (shouldContinue) {
+
+}
+
+// bad
+if (!options || (options.continue && options.hasWon)) {
+
+}
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+### Negative conditional expressions
+
+Positive conditional expressions are generally easier to follow than negative ones. Try to flip the logic so that the positive case is considered first:
+
+```js
```

Comment:
I think these examples are too abstract and could benefit from a more concrete example with real variables